README.md

```diff
@@ -1,23 +1,178 @@
-||||||||[access tiller endpoint](/controls/accesstillerendpoint.json)|||||
-||||||||[CoreDNS poisoning](/controls/coreDNSpoisoning.json)|||||
-||||||||ARP and IP spoofing|||||
+Rule - a single specific test
 
+---
 
-## Testing
-See [testing](testrunner/README.md)
\ No newline at end of file
+
+## **How to**
+
+### **Add a framework** 
+
+Add a frameworkName.json file in frameworks directory
+
+Example of a framework:
+```
+{
+    "name": "DevOpsBest",
+    "description": "This framework is recommended for use by devops.",
+    "attributes": {
+      "armoBuiltin": true
+    },
+    "controlsNames": [
+        "Naked PODs",
+        "Containers mounting Docker socket",
+        "Image pull policy on latest tag",
+        "Label usage for resources",
+        "K8s common labels usage",
+        "Pods in default namespace",
+        "Container hostPort",
+        "Resources CPU limit and request",
+        "Resources memory limit and request",
+        "Configured liveness probe",
+        "Configured readiness probe"
+    ]
+}
+```
+* Attribute "armoBuiltin": true - mandatory for armo rules. Only ARMO team members are authorized to create builtin objects.
+* controlNames - List of controls to run, must be exact name. Copy-paste to be sure.
+
+### **Add a control**
+
+Add controlName.json file in controls directory.
+
+Example of a control:
+```
+{
+    "name": "Pods in default namespace",
+    "attributes": {
+        "armoBuiltin": true
+    },
+    "description": "It is recommended to avoid running PODs in cluster without explicit namespace assignment. This control identifies all the PODs running in the default namespace.",
+    "remediation": "Create necessary namespaces and move all the PODs from default namespace there.",
+    "rulesNames": [
+        "pods-in-default-namespace"
+    ],
+    "long_description": "It is recommended to avoid running PODs in cluster without explicit namespace assignment. This may lead to wrong capabilities and permissions assignment and potential compromises. This control identifies all the PODs running in the default namespace.",
+    "test": "Check that there are no pods in the 'default' namespace",
+    "id": "C-0061",
+    "controlID": "C-0061",
+    "baseScore": 3
+}
+```
+* Attribute "armoBuiltin": true - mandatory for armo rules. Only ARMO team members are authorized to create builtin objects.
+* rulesNames -  List of rules to run, must be exact name. Copy-paste to be sure.
+
+* "long_description", "test" and other CIS fields are used mainly in [docs](https://hub.armosec.io/docs)
+
+* See [control go struct](https://github.com/kubescape/opa-utils/blob/fc41353af5c67221dead93c353b48e88f02a9fe4/reporthandling/datastructures.go#L56) for more control fields
```

Comment:
This leads to a specific commit. Is there a way to make it to the latest commit always?

Reply:
yes. I'll fix